pkg/steps/multi_stage/gen.go

```diff
@@ -124,7 +124,13 @@ func (s *multiStageTestStep) generatePods(
 		delete(pod.Labels, base_steps.ProwJobIdLabel)
 		pod.Annotations[base_steps.AnnotationSaveContainerLogs] = "true"
 		pod.Labels[MultiStageTestLabel] = s.name
-		pod.Spec.ServiceAccountName = s.name
+		if step.NoKubeconfig == nil || !*step.NoKubeconfig {
```

Comment:
can we just create a variable like `noKubeconfig := step.NoKubeconfig != nil && *step.NoKubeconfig` so that we don't have to keep doing this same logic everywhere?

Reply:
sounds good - done